### DISCUSSION TITLE
remap german umlauts

### DISCUSSION THREAD

**Discussion by pilger** [helix-editor/helix#2618] [Q&A]
is there a trick to remap german umlauts (öäü)? for some reason i can't make it work.

**Comment by getreu** [comment]
These should be mapped on operating system level:
Example: [Eurkey](https://blog.getreu.net/tags/eurkey/)

**Reply by YoRHa-A5** [reply]
Wouldn't this remap the whole keyboard? What's the point if the physical keyboard still has the european layout?

**Comment by YoRHa-A5** [comment]
> is there a trick to remap german umlauts (öäü)? for some reason i can't make it work.

```toml
[keys.normal]
"ö" = "collapse_selection" 
```

Putting it inside quotes seems to work.
